### PR TITLE
FM2-344: Improve Integration Tests for group resource

### PR DESCRIPTION
**Pull request by Medhavi-16**
## Issue I worked on

see https://issues.openmrs.org/browse/FM2-344

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by Medhavi-16** [comment]
@corneliouzbett  @ibacher I added the integration test modules for platform version 2.1. PTAL.

**Comment by HerbertYiga** [comment]
add the ticket Id to your commit as guided here https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by Medhavi-16** [comment]
> add the ticket Id to your commit as guided here https://wiki.openmrs.org/display/docs/Pull+Request+Tips

I have added the ticket id in the PR title as mentioned in the link. I didn't see anything about the commit message though.

File: pom.xml [hunk 1]
```diff
@@ -42,6 +42,7 @@
         <module>omod</module>
         <module>integration-tests</module>
         <module>integration-tests-2.2</module>
+	    <module>integration-tests-2.1</module>
```
Comment (ibacher) [inline pom.xml:45]:
```suggestion
        <module>integration-tests-2.1</module>
        <module>integration-tests-2.2</module>
```

**Review by ibacher — APPROVED** [review]

**Comment by ibacher** [comment]
@Medhavi-16 Thanks for this!

**Pull request merged**